AjayKumarMK/assignment2/django_blog/blog/templates/blog.html

```diff
@@ -0,0 +1,40 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Django Blog - Home</title>
+    <link rel="stylesheet" type="text/css" href="{% static 'main.css' %}">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+</head>
+<body>
+    <div class="header">
+        <h1 href="#">My Blog</h1>
+    </div>
+    <div class="new-body">
+        <div class="card" >
+          <div>
+            <a href="create/" class="btn btn-outline-primary">Create New Blog</a>
+            <a href="/signout" class="btn btn-outline-danger" style="float: right;">Logout</a>
+          </div>
+        </div>
+        
+        {% for blogs in blogs %}
```

Comment:
Naming convention is not right , can you change it to for blog in blogs?
Also change it down below where blogs.property has been used